docs/reference/adaptors.rst

```diff
@@ -104,6 +104,27 @@ Adaptors
 ..  function::
     auto drop(sequence auto seq, std::integral auto count) -> sequence auto;
 
+    Given a sequence :var:`seq` and a non-negative integral value :var:`count`, returns a new sequence which skips the first :var:`count` elements of :var:`seq`.
+
+    The returned sequence is always a :concept:`sequence`, with the same category of :var:`seq`. If :var:`seq` has fewer than :var:`count` elements, the adapted sequence is out of range.
```

Comment:
> The returned sequence is always a :concept:`sequence`, with the same category of :var:`seq`

I think that elsewhere I say "the returned sequence has the same capabilities as :var:`seq`".

> If :var:`seq` has fewer than :var:`count` elements, the adapted sequence is out of range.

"Out of range" is not quite right: it would be better to say "..., the returned sequence is empty."

Reply:
Thanks for corrections. Originally, I want to write "the returned sequence is empty", but this example not working:
https://godbolt.org/z/z88srjvx5

Reply:
Oh dear, that's really bad :(

Drop and take were two of the first adaptors I wrote for Flux, it looks like I didn't do a very good job!